forem/forem
# Text overflow on profile preview cards

## What type of PR is this? (check all applicable)


## Description

This PR fixes a bug with overflowing text on profile preview cards.

## Related Tickets & Documents

- Closes #14971

## QA Instructions, Screenshots, Recordings

Add a long word (like a URL) to a summary and then open it in a feed card like shown. It should break if it's too long.

Before:

![image](https://user-images.githubusercontent.com/108287/154960567-54f10f19-3f7f-42d8-8141-1c2a37335808.png)

After:

![image](https://user-images.githubusercontent.com/108287/154960515-8611b9c8-0ba2-40fe-a7a7-4bfc2594d2f2.png)


### UI accessibility concerns?

Nope

## Added/updated tests?

Nope

## [Forem core team only] How will this change be communicated?

It won't.
